examples/large_files/test/manual_tests.ts

```diff
@@ -0,0 +1,61 @@
+import { describe } from '@jest/globals';
+import { expect, it, please, Test } from 'azle/test/jest';
+import { execSync } from 'child_process';
+import { join } from 'path';
+
+import { Unit } from '../../../scripts/file_generator';
+import { generateTestFileOfSize } from './generate_test_files';
+import { getAutoGeneratedFileName, verifyUpload } from './tests';
+
+export function manualTests(origin: string): Test {
+    const autoGenManualUploadTests: [number, Unit][] = [[150, 'MiB']];
```

Comment:
Rename similar to other renames, not ending in Tests, they are something besides tests.

Reply:
done

Reply:
Plural

Reply:
done